src/hu_stuff.c

```diff
@@ -1043,6 +1036,9 @@ static void HU_UpdateCrosshair(void)
     ammotype_t ammo = weaponinfo[plr->readyweapon].ammo;
     fixed_t range = (ammo == am_noammo) ? MELEERANGE : 16*64*FRACUNIT;
     boolean intercepts_overflow_enabled = overflow[emu_intercepts].enabled;
+    
+    linetarget = NULL;
```

Comment:
Something I noticed: it appears that the crash I was experiencing on Nugget Doom - the one fixed by means of the `gamestate == wipegamestate` check - does not happen here, but I guess we can keep this regardless.

Reply:
Yes, please.